src/pages/verify/guides/faq.mdx

```diff
@@ -19,4 +19,10 @@ The `Unknown Access Code` error can happen for one of two reasons when trying to
 
 If you are seeing this error in production but it worked in test, make sure that your production application and test application have the exact same OIDC settings.
 
-You can use the copy application feature from test to production to ensure the settings are the same.
\ No newline at end of file
+You can use the copy application feature from test to production to ensure the settings are the same.
+
+## MitID Core Client error
+
+If you encounter the `mitid_core_client_error` while running test logins, it indicates issues with the MitID PP environment. 
+Please note that the MitID PP environment can occasionally experience instability, and MitID does not provide SLAs or status updates for this environment.
+Criipto does not have control over the MitID Test Tools.
```

Comment:
Mention that if they are seeing issues with an app simulator user, they can try registering a TOTP authenticator instead.

Reply:
Ok, good to know, thanks.

Reply:
![image](https://github.com/user-attachments/assets/e3a74593-8d19-4da0-a482-50b20ddc9525)
May want individual paragraphs.

Reply:
Updated